SkillChat.Client/Utils/SetMessagesToClipboard.cs

```diff
@@ -0,0 +1,14 @@
+﻿using SkillChat.Client.ViewModel.Interfaces;
+using Avalonia;
+using Avalonia.Input.Platform;
+
+namespace SkillChat.Client.Utils
+{
+    class SetMessagesToClipboard : IClipboardMessage
```

Comment:
Мне кажется не логичным message в названии этого интерфейса
```suggestion
    class SetMessagesToClipboard : IClipboard
```